features/ALB999_rail_alignment_value_constraints.feature

```diff
@@ -0,0 +1,17 @@
+@implementer-agreement
+@ALB
+Feature: Rail Alignment value constraints
+
+  Background: Rail alignment
+  
+    Given An IfcAlignment
+      And A relationship IfcRelReferencedInSpatialStructure to IfcAlignment from IfcRail
+    
+  Scenario: Agreement on allowed values for horizontal segment types
+    
+    Given A relationship IfcRelNests from IfcAlignment to IfcAlignmentHorizontal and following that
+    Given A relationship IfcRelNests from IfcAlignmentHorizontal to IfcAlignmentSegment and following that
```

Comment:
What is `and following that`?

Reply:
Good question, I've updated the PR description. You can find the answer under `traversal`. In short it means that, in the first case without `and following that` the `a relationship ...` statement checks whether such a relationship exists, but keeps the context on the previously active node (alignment). In the second case we actually descend according to the nesting structure, so that the attribute DesignParameters is read from the IfcAlignmentSegment.